postgres_storage.go

```diff
@@ -0,0 +1,168 @@
+package main
+
+import (
+	"fmt"
+	"github.com/jackc/pgconn"
+	_ "github.com/jackc/pgx/v4/stdlib"
+	"github.com/jmoiron/sqlx"
+	uuid "github.com/satori/go.uuid"
+	"strings"
+)
+
+type PostgresStorage struct {
+	db *sqlx.DB
+}
+
+const connectionString = "postgres://postgres:postgres@localhost:5432/wasfaty?sslmode=disable"
+
+func NewPostgresStorage() (*PostgresStorage, error) {
+	db, err := sqlx.Connect("pgx", connectionString)
+	if err != nil {
+		return nil, err
+	}
+	return &PostgresStorage{db}, nil
+}
+
+func (s *PostgresStorage) GetAll() []*Person {
+	var pp []*Person
+
+	err := s.db.Select(&pp, `SELECT * FROM person`)
+	if err != nil {
+		return nil
+	}
+
+	for _, p := range pp {
+		var cc []*Communication
+		err = s.db.Select(&cc, fmt.Sprintf(`SELECT Value
+			FROM Communication
+			WHERE PersonId = '%s'`, p.ID.String()))
```

Comment:
you should use escaped parameters wherever it is possible https://jmoiron.github.io/sqlx/#getAndSelect
manual formatting sql strings is potential sql injection vulnerability.